xikolo-ios/ViewControllers/Courses/CourseContentTableViewController.swift

```diff
@@ -79,6 +79,19 @@ class CourseContentTableViewController: UITableViewController {
         }
     }
 
+    func showProctoringDialog() {
+        let title = NSLocalizedString("This item needs to be proctored", comment: "Shown in proctoring dialog")
+        let message = NSLocalizedString("You can't do this on your mobile device.", comment: "Shown in proctoring dialog")
```

Comment:
@jprberlin @rhlff I thought you may want to add a more elaborate explanation or hints on what to do here. I will add the translation when the message is ready. Or is this enough?